chess/core/Cargo.toml

```diff
@@ -4,5 +4,5 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-risc0-zkp = { version = "0.11", default-features = false }
+risc0-zkp = { git = "https://github.com/risc0/risc0.git", rev = "65fb3871e3ef468be0a306d9f1adf5631d943b46", default-features = false }
```

Comment:
Does this blob indicate that we're using `0.12-rc-2`?

Reply:
No, it means we're using a specific intermediate commit (`65fb3871e3ef468be0a306d9f1adf5631d943b46`). It's a bit of a relic of me starting [tzerrell/update-r0-version](https://github.com/risc0/risc0-rust-examples/tree/tzerrell/update-r0-version) before `0.12.0-rc.1` was out. Everything will need to be updated to `0.12.0` once the actual release is out, but it's not yet, so everything landing in [tzerrell/update-r0-version](https://github.com/risc0/risc0-rust-examples/tree/tzerrell/update-r0-version) (like this is) can use an intermediate commit if it wants and we can batch update.

Switching to 0.12.0-rc.2 now that it's out would probably be a little cleaner, but even that would need to be updated so IMO let's not worry about it except on new code.